hangups/conversation.py

```diff
@@ -548,6 +637,69 @@ async def set_notification_level(self, level):
             )
         )
 
+    async def modify_otr_status(self, off_record):
+        """Set the OTR mode of this conversation.
+
+        Args:
+            off_record: ``True`` to disable history, or ``False`` to enable it.
+
+        Raises:
+            NetworkError: If the request fails.
+
+        Returns:
+            :class:`.ConversationEvent` representing the change.
+        """
+        if off_record:
+            status = hangouts_pb2.OFF_THE_RECORD_STATUS_OFF_THE_RECORD
+        else:
+            status = hangouts_pb2.OFF_THE_RECORD_STATUS_ON_THE_RECORD
+        try:
+            response = await self._client.modify_otr_status(
+                hangouts_pb2.ModifyOTRStatusRequest(
+                    request_header=self._client.get_request_header(),
+                    otr_status=status,
+                    event_request_header=self._get_event_request_header(),
+                )
+            )
+            return self._wrap_event(response.created_event)
+        except exceptions.NetworkError as e:
+            logger.warning('Failed to set OTR mode: {}'.format(e))
+            raise
+
+    async def set_group_link_sharing(self, enabled):
```

Comment:
I think `set_group_link_sharing_enabled` reads better, although it's a little long.

Reply:
Probably worth it for consistency with the underlying client call.